LondonBritishLibrary/add/BLadd18993.xml

```diff
@@ -104,18 +104,27 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     <material key="parchment"/>
                                 </support>
                                 <extent>
-                                    <measure unit="leaf">49</measure>
+                                    <measure unit="leaf" quantity="57">4+1+49+3</measure>
```

Comment:
Why is here the string 4+1+49+3? Not 5+49+3?

Reply:
i-iv are modern flyleaves, v probably originally belonged to the manuscript. But it can be changed, in these records this notation mostly reflects the foliation, not the "real" flyleaves.

Reply:
I would say, it reflect the foliation first (breaks within the main block are not reflected by foliation, too).

Reply:
so better just 5+49+3, you mean?

Reply:
@DenisNosnitsin1970 ? :)